### PR TITLE
Add new filter based on jmespath for 6.0 version

### PR DESCRIPTION
**Pull request by snifbr**
This add to cloudmonkey CLI the abilities to make a JMESPATH filter with following changes in version 6.0:

- Add new argument 'query'.
- Import jmespath module.

OBS: I used the same way to print response like "filter=", then all the output formats are compatible with this new feature. I hope this will be useful for a first version of this feature.

Some examples:
Screenshot from 2019-07-18 17-34-46

Screenshot from 2019-07-18 17-36-34

### CODE REVIEW

File: cli/exec.go [hunk 1]
```diff
@@ -57,6 +57,8 @@ func ExecCmd(args []string) error {
 		return nil
 	}
 
+	// fmt.Printf("args0 => %s", args[0])
```
Comment (rhtyd) [inline cli/exec.go]: Remove commented code

**Comment by rhtyd** [comment]
Thanks @snifbr I just saw the PR, I'll test it and get back to review the PR soon.

**Comment by romain-dartigues** [comment]
@rohityadavcloud < is there some hope to see some of this land in the main?
Or has it been supplanted by #114?

**Comment by rohityadavcloud** [comment]
@romain-dartigues I think the pipe to `jq` has always worked from terminal and now with #114  in interpreter mode so I think there's no urgency for this. I don't maintain/manage cmk these days, perhaps our RM for the next release cc @borisstoyanov can advise.

**Comment by borisstoyanov** [comment]
@snifbr could you please resolve the conflicts

**Comment by rohityadavcloud** [comment]
Thanks @snifbr for your PR and hard work. As I've noted on https://github.com/apache/cloudstack-cloudmonkey/issues/45 I think we would prefer that cmk continues to be a simple CLI and other tools such as jq can be used to add in additional filtering mechanisms. Any additional such features are better be supported in the cloudstack mgmt server API.

**Pull request closed without merging**